deblur/test/data/readme.txt

```diff
@@ -0,0 +1,7 @@
+70_otus.fasta - the 70% representative set from greengenes 13.5
+
+seqs_s1.fa - a simulated dataset with 100 reads derived from simset.s1.fasta using art
+seqs_s2.fa - a simulated dataset with 200 reads derived from simset.s2.fasta using grinder, with 10% chimeras, and additionally 35 phix reads
```

Comment:
added code to generate the simulated datasets to the readme

On Thu, Mar 17, 2016 at 4:43 AM, Evguenia Kopylova <notifications@github.com
> wrote:

> In deblur/test/data/readme.txt
> <https://github.com/biocore/deblur/pull/32#discussion_r56491062>:
>
> > @@ -0,0 +1,7 @@
> > +70_otus.fasta - the 70% representative set from greengenes 13.5
> > +
> > +seqs_s1.fa - a simulated dataset with 100 reads derived from simset.s1.fasta using art
> > +seqs_s2.fa - a simulated dataset with 200 reads derived from simset.s2.fasta using grinder, with 10% chimeras, and additionally 35 phix reads
>
> is there still the more detailed explanation of how these were created
> (e.g. actual ART commands and expected sequences that were part of the
> comment in the code associated to these datasets)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/deblur/pull/32/files/069a0cd16c523f104f10937cd3dd5fd5715c57bb#r56491062>
>